### PR TITLE
Make it clearer to users that they still need to submit the application

### PR DESCRIPTION
**Pull request by timpaul**
Drop-out rates on this page are about 50% right now. We'd like to try adjusting the content on the page and see if that has any significant effect. We will need to know when this change has gone live, so we can accurately measure the effect on completion rates.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -2,9 +2,10 @@ en:
   privacy_notice:
     label: Privacy
   check_your_answers:
-    title: Check your answers before sending your application
+    title: Are you ready to send your application?
+    hint: Check your answers first
```
Comment (timpaul) [inline config/locales/en.yml]:
```suggestion
    hint: Check your answers first.
```

Comment (36degrees) [inline config/locales/en.yml]: I think that this will need to be added to the view as well before it has any effect.

File: config/locales/en.yml [hunk 2]
```diff
@@ -2,9 +2,10 @@ en:
   privacy_notice:
     label: Privacy
   check_your_answers:
-    title: Check your answers before sending your application
+    title: Are you ready to send your application?
+    hint: Check your answers first.
```
Comment (timpaul) [inline config/locales/en.yml:6]: This is intended to appear immediately below the H1. Are any changes required to other files to make this work?

File: config/locales/en.yml (hunk 1 above)
Comment (timpaul) [reply config/locales/en.yml]: @leenagupte - this is beyond my knowledge :-/ Could someone else take a look please?

Comment (leenagupte) [reply config/locales/en.yml]: Looking now 👀

Comment (leenagupte) [reply config/locales/en.yml]: https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form/pull/167

**Pull request closed without merging**